src/core/js/device.js

```diff
@@ -44,9 +44,22 @@ define(function(require) {
             : window.innerWidth || $window.width();
     }
 
+    function getScreenHeight() {
+        return isAppleDevice()
+            ? getAppleScreenHeight()
+            : window.innerHeight || $window.height();
+    }
+
+    function getScreenOrientation() {
+        return (Adapt.device.screenWidth >= Adapt.device.screenHeight) ? 'landscape' : 'portrait';
```

Comment:
I seem to remember the screen height doesn't change when the orientation changes on iOS. So you'd always have to use the window.innerWidth to calculate the orientation. I'll see if that applies here a bit later and find some references if applicable.